app/components/atoms/T/index.js

```diff
@@ -6,15 +6,18 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import { compose } from 'redux';
-import { injectIntl } from 'react-intl';
 import { Text } from 'react-native';
 
-const T = ({ intl, id, values, style, text, ...otherProps }) => (
-  <Text testID="t" style={style} {...otherProps}>
-    {id ? intl.formatMessage({ id }, { ...values }) : text}
-  </Text>
-);
+import { useTranslation } from 'react-i18next';
+
+const T = ({ intl, id, values, style, text, ...otherProps }) => {
+  const { t } = useTranslation();
+  return (
+    <Text testID="t" style={style} {...otherProps}>
+      {id ? t(id, { ...values }) : text}
```

Comment:
Can we please add a conditional operator function here

Reply:
done